client/src/pages/ProfileDetails/ProfileDetails.tsx

```diff
@@ -49,11 +92,25 @@ const ProfileDetails = (): JSX.Element => {
             />
           </Box>
         </CardContent>
+        <CardContent className={classes.reviewsCardContent}>
+          <Typography className={classes.reviewsHeader}>reviews ({reviews.length})</Typography>
+          <Box display="flex" flexDirection="column" alignItems="center" justifyContent="center" paddingTop="10px">
+            {reviews.map((review) => {
+              return (
+                <CardContent key={review.clientId + review.sitterId} className={classes.review}>
```

Comment:
Why don't we consider using something unique (id of a review document) because if an owner leaves more than one review - the respective component will have identical keys